src/config/logger.ts

```diff
@@ -91,6 +91,11 @@ export const getLogger = (name: string): LoggerWithTarget => {
 	return logger.child({ name });
```

Comment:
Adding the feature flag here and returning safe or unsafe logger was considered.
Reasons against... 
- would convert this function to async :( its consumed everywhere so would be a headache(onflagchange event might be an option though...)
- When we log the confidential data we need to hash for safe and not hash for unsafe so its not quite as simple as a gate to the environment there is unique logic likely per log event.